src/vs/workbench/contrib/terminal/browser/media/shellIntegration-bash.sh

```diff
@@ -59,7 +59,8 @@ __vsc_update_cwd() {
 
 __vsc_command_output_start() {
 	builtin printf "\033]633;C\007"
-	builtin printf "\033]633;E;$__vsc_current_command\007"
+	# Send command line, escaping printf format chars %
+	builtin printf "\033]633;E;$(echo $__vsc_current_command | sed s/%/%%/g)\007"
```

Comment:
We don't have to escape if `$__vsc_current_command` is passed as a separate parameter to `printf`, avoiding the need for the sub-shell and sed, like this:

```sh
builtin printf "\033]633;E;%s\007" "$__vsc_current_command"
```